.github/workflows/release-master.yml

```diff
@@ -10,11 +10,6 @@ on:
       # Exclude all pre-releases
       - "!v*[0-9]+.[0-9]+.[0-9]+-*"
   workflow_dispatch:
```

Comment:
Could probably remove this, given we only want to run the release upon tag

Reply:
This was added having in mind that maybe we can use it for rollbacks. Not 100% sure how is this handled now. But maybe we can have a different pipeline for it if needed, that does not need to rebuild everything

Reply:
Yeah we can use the existing deploy-cloud job to push out any version we want so I'd say it's safe / desirable to remove the dispatch